tsutils/json_utils.py

```diff
@@ -46,7 +46,7 @@ def validate_json(fp):
     try:
         json.load(open(fp))
         return True
-    except:
+    except Exception:
```

Comment:
Can you catch a more specific exception or set of exceptions here? if not it's ok